packages/core/core/src/public/Asset.js

```diff
@@ -87,6 +88,10 @@ class BaseAsset {
     return this.#asset.value.filePath;
   }
 
+  get query(): QueryParameters {
```

Comment:
Should transformers be able to add additional data to the query?

Only example I can think of is html-transformer that would add width and height from srcset to the image asset, but that's probably something with addDependency anyway